packages/MainUI/components/Nav/Profile.tsx

```diff
@@ -72,6 +72,7 @@ const ProfileWrapper = (props: ProfileWrapperProps) => {
   return (
     <ProfileModal
       {...props}
+      token={token}
```

Comment:
It would be nice if you implement the approach I suggested in order to remove this prop